src/argus_htmx/templatetags/argus_htmx.py

```diff
@@ -11,4 +11,22 @@ def tagvalues(incident, key) -> list:
     There can be multiple tags with the same key
     """
     tags = incident.deprecated_tags
-    return [str(tag.value) for tag in tags]
+    return [str(tag.value) for tag in tags if tag.key == key]
+
+
+@register.filter
+def is_acked_by(incident, context=None) -> bool:
+    """
+    If ack_group is None, return True if incident is acked by any group
+    If ack_group is not None, return True if incident is acked by given ack_group
+    """
+
+    if not context:
+        is_acked = incident.acked
+    else:
+        if context.ack is None:
+            is_acked = incident.acked
+        else:
+            is_acked = context.ack in incident.acks.active().group_names()
+
+    return is_acked
```

Comment:
Sikt just needs a template that checks if `incident.acked` is True.

Geant needs a template that checks if `incident.acked` is True and also checks if it belongs to a specific group.

This filter doesn't work well for either, it is hard to understand and over-abstracted. It would be better in this one case to either *not* have this filter and expect Geant to make their own or make a filter that returns True if this incident is acked and was acked by a user of a specific group.